src/shared/config.ts

```diff
@@ -39,6 +39,7 @@ export interface ServiceConfig {
   PARTICIPANT_ID: string;
 
   // ../lib/config.
+  ENVIRONMENT: string;
```

Comment:
Hmm I don't like generic, superpowered config name like `ENVIRONMENT`. Is there a better one you can think of? Or if this is just to configure the database connection, we should maybe rely on users mounting their own knex file like we do in other projects.